### ISSUE TITLE
Cannot add the versioning task to VSTS - 

### ISSUE THREAD

**Issue by Aharonmi** [rfennell/vNextBuild#149]
### VSTS Extension you are using
Versioin### Where are you running it?
VSTS 

### Version of Extension/Task
The currently public version 

###Expected behavior and actual behavior :

Try to install the task through the marketplace . During the install we receive the following error 
  A task definition with id 'FD2EF332-9676-4E8F-8E95-B22BE61EC9DD', name '' has already been uploaded. To replace the existing definition, provide a task version which is greater than '1.18.3'.
The task was not previously installed and not showed in the extensions page 

### Steps to reproduce the problem
1. Add the package "Manifest Versioning Build Tasks" through the marketplace
2. Click confirm to install the package

result - the following error is displayed 
Try to install the task through the marketplace . During the install we receive the following error 
  A task definition with id 'FD2EF332-9676-4E8F-8E95-B22BE61EC9DD', name '' has already been uploaded. To replace the existing definition, provide a task version which is greater than '1.18.3'.

**Comment by rfennell** [comment]
I have just created a brand new VSTS account and installed this extension without any problems. So i think it is a local issue with your instance of VSTS

I have seen issues like this before, and it was caused by caching issues. As I remember the person fixed with one or more of the following

1. Logout of VSTS, login again (in a new browser session) and retry
1. Make sure you don't have an old version of the extension (or any of the tasks already installed). If you do remove them via the UI or the [TFX command line tool](https://github.com/Microsoft/tfs-cli)
1. If all this fails log a support call with MS they can check for catalogue issues on your instance. I had one of these in the past due to installing an extension at the same moment a VSTS update was rolling out which caused some dupications
